src/elements/content-sidebar/activity-feed/comment/Replies.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../../common/variables';
+
+.bcs-Replies {
+    margin: $bdl-grid-unit * 4.5 0;
+    padding-left: 40px;
```

Comment:
What is the `40px` value based on? Should this be using the grid, as well?

Reply:
We would be multiplying by 10 if we use the grid unit, which I have hesitencies about since it's such a large number. Maybe using a variable for this case would be better for readability since it's an uncommon value